addons/project/models/project.py

```diff
@@ -1842,7 +1849,7 @@ def write(self, vals):
             for task in self:
                 if not task.user_ids:
```

Comment:
Cannot we check if the task is **not** in the `task_ids_without_user_set` to only do the write if task had a user assigned before writing on it?

However, I guess the write is not called if the value is equivalent. 🤔 
I mean if the `date_assign` is already set to `False`, I guess the write is not called to set to False the `date_assign` field for that task.

What do you think? 🙂